src/projects/components/projects-page/parts/create-project/CreateProject.tsx

```diff
@@ -1,94 +1,103 @@
 import {Container, Group, rem, Stepper, Text} from '@mantine/core';
 import {useContext, useEffect, useState} from "react";
+import { useNavigate } from "react-router-dom";
 import {Dropzone} from "@mantine/dropzone";
 import {IconPhoto, IconUpload, IconX} from "@tabler/icons-react";
 import useAxios from "axios-hooks";
 import {ProjectForm} from "@/projects/components/parts/project-form/ProjectForm";
 import {Project} from "@/projects/entities/Project.ts";
 import { SettingsContext } from '@/core/utils/settingsContext.ts';
+import { notifications } from '@mantine/notifications';
 
 export function CreateProject() {
     const {local_backend} = useContext(SettingsContext);
-    const [active, setActive] = useState(0);
+    const navigate = useNavigate();
+    const [files, setFiles] = useState<File[]>([]);
+    const emptyProject: Project = {
```

Comment:
The form validates the project and before at that step the project was created already.

This was my solution about it and maybe we can set the name based on other actions or a random one each time.
But if there are better alternatives let me know. I just didn't want to touch too much the form itself.